docs/data/data-strategy.md

```diff
@@ -0,0 +1,87 @@
+# Data Strategy
+
+Mission: Help Thoth Tech Make Better Decisions, Faster.
```

Comment:
potential change to help with readability, and consistent with Documentation guide

```suggestion
**Mission:** Help Thoth Tech Make Better Decisions, Faster.
```